pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -999,6 +1004,7 @@ func TestPIDPressure(t *testing.T) {
 				nodeRef:                      nodeRef,
 				nodeConditionsLastObservedAt: nodeConditionsObservedAt{},
 				thresholdsFirstObservedAt:    thresholdsObservedAt{},
+				podsEvicted:                  sets.Set[string]{},
```

Comment:
Would it make sense to wrap this object creation into a `newManagerImpl` function to avoid duplication?